### PR TITLE
73 review accessibility

### PR DESCRIPTION
**Pull request by juanjoseortega-aircury**

### CODE REVIEW

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Where can I find the requirements for this ticket? Without them it cannot be reviewed.

File: _data/conf/main.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ search_with_uri: true
 #                Image Viewer
 ###########################################################
 # this enables image viewer
-image_viewer: true
+image_viewer: false
```
Comment (cristobaldiaz-aircury) [inline _data/conf/main.yml:95]: Would you mind to explain this change?

File: _data/conf/main.yml [hunk 2]
```diff
@@ -108,7 +108,7 @@ image_viewer_error_add_path: true
 #                Image lazy loader
 ###########################################################
 # this enables image lazy loader
-image_lazy_loader: true
+image_lazy_loader: false
```
Comment (cristobaldiaz-aircury) [inline _data/conf/main.yml:111]: Would you mind to explain this change?

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Some images hasn't changed in size, just the extension of the file. Is that correct?
image

Pending review for mobile and in all the pages. E.g. In projects:
image

Let's discuss it.

File: _data/conf/main.yml (hunk 1 above)
Comment (juanjoseortega-aircury) [reply _data/conf/main.yml:95]: This improves some of the performance to the web page.

File: _data/conf/main.yml (hunk 2 above)
Comment (juanjoseortega-aircury) [reply _data/conf/main.yml:111]: This improves some of the performance to the web page.

File: _data/conf/main.yml (hunk 1 above)
Comment (cristobaldiaz-aircury) [reply _data/conf/main.yml:95]: This comment should be in the code, so anybody can understand it while reading it.

File: _data/conf/main.yml (hunk 2 above)
Comment (cristobaldiaz-aircury) [reply _data/conf/main.yml:111]: This comment should be in the code, so anybody can understand it while reading it.

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Pending some small changes and the update of the requisites, to explain the issues that haven't been fixed and why.

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
There are some small improvements that could be introduced yet:
1. Contrast for the Cookies policy
image
And the close button:
image
2. List elements in the work with us page:
image
3. Create tickets for remaining work to improve performance and analyse SEO.
4. Fix conflicts with the main branch
5. Fix issue with images in the Spanish version of projects
image

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Broken emails:
image
Also in Spanish
image

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Links don't look right in Spanish:
image

After fixing this last thing, review if it's worth it to create a ticket to update the Spanish pages to the last version.

**Pull request merged**